keps/sig-node/4369-allow-special-characters-environment-variable/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: Allow special characters environment variable
+kep-number: 4369
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-node
+participating-sigs:
+  - sig-api-machinery
```

Comment:
Is this relevant to SIG Security?

Reply:
Based on the description, I assume this has nothing to do with SIG Security?https://github.com/kubernetes/community/tree/master/sig-security#security-special-interest-group

Reply:
Could be good to get a consultation - security holes are rarely where reasonable people expect them to be. :)

@IanColdwater @tabbysable ?

Reply:
ping @IanColdwater @tabbysable